src/code/z_lib.c

```diff
@@ -23,20 +25,56 @@ void Lib_MemSet(u8* dest, size_t len, u8 val) {
     // clang-format on
 }
 
+#ifdef DISABLE_LOOKUP_TABLE
+/**
+ * @param angle binang
+ * @return sinecos(sine, cosine)
+ */
+
```

Comment:
that blank space and the one at line 52 bothers me lol (especially the second one)

Reply:
done